MyWeight/Views/GradientView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  GradientView.swift
+//  MyWeight
+//
+//  Created by Bruno Mazzo on 11/6/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class GradientView: UIView {
+    
+    let gradientLayer = CAGradientLayer()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.configure()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.configure()
+    }
+    
+    func configure() {
```

Comment:
I like the `configure`, but we're using `setUp` for other classes.
What do you think to keep a standard? Any special reason for `configure`?